spec/system/candidate_interface/entering_details/degrees/candidate_editing_degrees_spec.rb

```diff
@@ -4,18 +4,20 @@
   include CandidateHelper
 
   before do
-    FeatureFlag.deactivate(:new_degree_flow)
+    allow(CycleTimetable).to receive(:current_year).and_return(CycleTimetable.current_year)
```

Comment:
does this do something useful? looks like you're stubbing `CycleTimetable.current_year` with its default behaviour, but maybe i'm missing something.